package.json

```diff
@@ -897,7 +897,7 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.10.0",
-    "@tony.ganchev/eslint-plugin-header": "^3.1.2",
+    "eslint-plugin-header": "github:tonyganchev/eslint-plugin-header#36161b3282e088ad2f19657ab0e377651b6886c2",
```

Comment:
Aren't these two the same thing? And the package from the npm is versioned.

Reply:
Yes they are! But I would advise to keep using the git URL. That user might re-publish their v3.1.2 version and introduce anything. We should not trust the NPM package. However, the git URL acts as a sha256sum

Reply:
So both should be good as long as the extreme case doesn't happen such as repository is deleted or the npm package is unpublished.